Tasks/UsePythonV1/version.ts

```diff
@@ -0,0 +1,27 @@
+/** Convert versions like `3.8-dev` to a version like `>= 3.8.0-a0`. */
+export function desugarDevVersion(version: string | undefined): string {
+    if (!version) {
+        throw('Version is required.');
```

Comment:
This should just be banned in the type signature.  The caller needs to make sure it's defined before calling the function.